### PR TITLE
CHK-1376-new-wallet-domain-common

### PR DESCRIPTION
**Pull request by Fernando-Granato**
### List of changes
Added new domain for wallet. 
For now the configuration provides only common with cosmos resource.

### Motivation and context
The change was made to allow the generation of a new domain common for the wallet project.

### Type of changes

- [x] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/domains/wallet-common/env/weu-dev/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  public_network_access_enabled     = true
+  is_virtual_network_filter_enabled = false
+
+  backup_continuous_enabled = false
+
+}
+
+cidr_subnet_cosmosdb_wallet = ["10.1.153.0/24"]
+cidr_subnet_redis_wallet    = ["10.1.148.0/24"]
+cidr_subnet_storage_wallet  = ["10.1.154.0/24"]
```
Comment (pietro-tota) [inline src/domains/wallet-common/env/weu-dev/terraform.tfvars:57]: Maybe those have to differs from the one from ecommerce domain?

File: src/domains/wallet-common/env/weu-uat/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  public_network_access_enabled     = false
+  is_virtual_network_filter_enabled = true
+
+  backup_continuous_enabled = false
+
+}
+
+cidr_subnet_cosmosdb_wallet = ["10.1.153.0/24"]
+cidr_subnet_redis_wallet    = ["10.1.148.0/24"]
+cidr_subnet_storage_wallet  = ["10.1.154.0/24"]
```
Comment (pietro-tota) [inline src/domains/wallet-common/env/weu-uat/terraform.tfvars:57]: Same as above

**Review by pietro-tota — COMMENTED** [review]
LGTM, just some comment

File: src/domains/wallet-common/env/weu-dev/terraform.tfvars (hunk 1 above)
Comment (Fernando-Granato) [reply src/domains/wallet-common/env/weu-dev/terraform.tfvars:57]: No should be changed with those specific to the new domain

File: src/domains/wallet-common/env/weu-uat/terraform.tfvars (hunk 2 above)
Comment (Fernando-Granato) [reply src/domains/wallet-common/env/weu-uat/terraform.tfvars:57]: No should be changed with those specific to the new domain

File: src/domains/wallet-common/env/weu-dev/terraform.tfvars (hunk 1 above)
Comment (infantesimone) [reply src/domains/wallet-common/env/weu-dev/terraform.tfvars]: subnet updated

File: src/domains/wallet-common/env/weu-uat/terraform.tfvars (hunk 2 above)
Comment (infantesimone) [reply src/domains/wallet-common/env/weu-uat/terraform.tfvars]: ditto

**Review by pietro-tota — APPROVED** [review]

**Pull request merged**